lambda-load-old-data/lambda_load_s3.py

```diff
@@ -0,0 +1,121 @@
+"""Script that selects data from previous day in RDS database and inserts it into lnhm_archive.csv file in S3 bucket"""
+
+import csv
+from os import environ, path
+
+from boto3 import client
+from botocore.exceptions import ClientError
+from dotenv import load_dotenv
+from sqlalchemy import create_engine, sql, engine
+
+
+def get_database_connection() -> engine.base.Connection:
```

Comment:
Maybe you can pass in config as an argument to this and the connecting to the client function, similar to what Zander suggested yesterday?